### DISCUSSION TITLE
Unable to Re-Verify Student Discount

### DISCUSSION THREAD

**Discussion by eiiot** [community/community#54179] [GitHub Education]
Hi everyone,

I've tried about 20 times to re-verify my student status, but it seems like GitHub is just auto-denying requests. I keep getting the same response back:

> Unfortunately, we weren't able to approve your educational discount request for [@eiiot](https://github.com/eiiot).
> 
> What happened?
>
> Common verification problems and suggested solutions are documented here: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved.

The main issue looks like something with my school email. When I select it on the menu, I get a message that says...

>  school-email@students.berkeley.net has already been used to obtain an academic discount on GitHub account(s) eiiot.

While this is true, I don't know if it prevents me from re-verifying with the same email. While I still have GitHub pro, I can't access my student benefits on sites like JetBrains because they think that I "haven't been granted a developer pack."

Other things I've tried:

- using a different browser, computer, OS, for the application
- using different documents (screenshot of schedule, transcript, etc.)
- writing down my school details and taking a photo and using that as the "ID"

I would appreciate any help. Other threads on this forum with the same problem have been ignored, and GitHub support auto-closed my ticket. As a student, I rely on GitHub and the student developer pack to do my job, and help keep our campus running. It's really disappointing to see GitHub's lack of transparency and all-around uselessness in this matter.

Thanks

**Comment by AliAmmarNaseem** [comment]
I am receiving same error, let me know if you found any solution, Thanks.

**Comment by samus-aran** [comment]
Hi @eiiot 👋 

Can you check if your location settings have the right permissions.

- When asked, please enable the application to discover your location from the browser pop-up. Since settings for every browser are different, if you are having difficulty enabling us to see your location you should seek assistance from your browser's support pages.
- If you are still having difficulties enabling your location check out [this discussion on assessing your hardware or operating system's settings.](https://github.com/orgs/community/discussions/42246)

You may need to do a hard refresh of the application page after you've enabled your browser location permissions. Let us know how you get on 👍

**Reply by eiiot** [reply]
The system configuration is correct for allowing Firefox (and Chrome, for that matter) to access my location, and I've given GitHub my location permission within each browser. Any other ideas?

**Comment by syedsameerpasha** [comment]
Hlo I have taken a course of full stack web development from Devtown,,The react js project that is bookmyshow clone is in older version and I am facing lot of issues to do that because it is not working plz solve this or provide vedios with updated version
